libmscore/segment.cpp

```diff
@@ -1062,6 +1062,20 @@ bool Segment::hasElements(int minTrack, int maxTrack) const
       return false;
       }
 
+//---------------------------------------------------------
+//   allElementsInvisible
+///  return true if all elements in the segment are invisible
+//---------------------------------------------------------
+
+bool Segment::allElementsInvisible() const
+      {
+      for (Element* e : _elist) {
+            if (e && e->visible())
```

Comment:
Without thinking this totally through or testing it, it seems this would cause invisible rests to now be ignored?  If so, this seems like it would break existing scores that rely on this to achieve particular spacing effects (eg, direct proportional spacing, equal width measures, etc).